pkg/fftypes/batch.go

```diff
@@ -30,7 +30,7 @@ type Batch struct {
 	Namespace string      `json:"namespace"`
 	Type      MessageType `json:"type"`
 	Node      *UUID       `json:"node,omitempty"`
-	Identity
+	SignerRef
```

Comment:
Per previous comments, this code level spelling change is not visible on the external JSON.